eeg_load_xdf.m

```diff
@@ -81,8 +81,9 @@
         chn = stream.info.desc.channels.channel{c};
         if isfield(chn,'label')
             chanlocs(c).labels = chn.label; end            
-        if isfield(chn,'type')
-            chanlocs(c).type = chn.type; end
+        if isfield(chn,'type') && strcmpi(chn.type, 'EEG')
+            chanlocs(c).type = chn.type; 
```

Comment:
Does chanlocs.type accept only the value 'EEG'? What about 'EMG', 'MEG', 'NIRS'?

Reply:
It should work as well. The 'EEG' is just in case types are not defined (it is as backup).

Arno